composer.json

```diff
@@ -34,6 +34,7 @@
         "spiral/roadrunner": "^2.11",
         "spiral/roadrunner-http": "^2.1",
         "spiral/roadrunner-worker": "^2.3",
+        "temporal/sdk": "^2.3",
```

Comment:
I am unsure it should be part of the runner. Using RoadRunner without temporal is pretty common.

Reply:
Either it should be moved to a separate package or to `suggest` section of `composer.json`.